src/ensembl/src/shared/components/badged-button/BadgedButton.test.tsx

```diff
@@ -26,66 +26,42 @@ const defaultProps = {
   badgeContent: faker.lorem.words()
 };
 
-const renderButton = (
-  ButtonComponent: React.FunctionComponent<any>,
-  props: any = defaultProps
-) => mount(<ButtonComponent {...props} />);
+const renderButton = (props: any = {}) =>
```

Comment:
We should try to avoid `any`s when at all possible. Here, the best way to specify the type of the props is to export the `Props` type from the `BadgedButton` component [here](https://github.com/Ensembl/ensembl-client/blob/dev/src/ensembl/src/shared/components/badged-button/BadgedButton.tsx#L21-L25), then import it in this file (`import { Props as BadgedButtonProps } from './BadgedButton`), and declare the props in this function to be `Partial<BadgedButtonProps>`.